environment.yml

```diff
@@ -21,8 +21,8 @@ dependencies:
   - pillow
   - pip
   - pip:
+    - "pyknos@git+ssh://git@github.com/mackelab/pyknos.git"
```

Comment:
if a user doesnt have ssh set up, this will fail, no?  This is probably why the tests in github actions fail during installation.